net.sonic_pi.SonicPi.json

```diff
@@ -205,16 +234,345 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://github.com/sonic-pi-net/sonic-pi/archive/v3.3.1.tar.gz",
-                    "sha256": "6aab8e22daff1832b7af8221d7d124e4ae0adeaf1c726d8f62a8d3d57bd27d9c"
+                    "url": "https://github.com/sonic-pi-net/sonic-pi/archive/refs/tags/v4.2.0.tar.gz",
+                    "sha256": "22376b5acf528582c6b8f43a6c53df368775ac4cc7b1136fdfc073c05d2ad56d"
                 },
                 {
-                    "type": "patch",
-                    "path": "sonic-pi-fixes.patch"
+                    "type": "archive",
+                    "url": "https://github.com/microsoft/vcpkg/archive/refs/tags/2022.08.15.tar.gz",
+                    "sha256": "89a3cb03aee1f081d4bf9fa9a610d6ae9275d92647099cf440f68fc40e980394",
+                    "dest": "app/vcpkg"
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/microsoft/vcpkg-tool/releases/download/2022-07-21/vcpkg-glibc",
```

Comment:
it's lovely to see that there is no mention of even the architecture the binary is meant to run on.

Note: we still build on aarch64.